lib/events/codes.go

```diff
@@ -96,6 +96,8 @@ const (
 	SessionUploadCode = "T2005I"
 	// SessionDataCode is the session data event code.
 	SessionDataCode = "T2006I"
+	// SessionRecording
+	SessionRecordingViewCode = "T2007I"
```

Comment:
I'm not sure if this event code is correct. I grouped it with other session events, but they're more related to a session in progress. Let me know if it should change.

Reply:
Seems okay to me. The important thing is that it ends with `I` and not `W` (which is reserved for error cases).

Also, don't forget the update the docs to document this new event. This can be done in a separate PR if you'd prefer.